### PR TITLE
Conversion of Gap3 from Mixer to Jekyll

### PR DESCRIPTION
**Pull request by User141**

### CODE REVIEW

File: Gap3/Packages3/mop.html [hunk 1]
```diff
@@ -17,8 +17,8 @@ <h3>
   Authors
 </h3>
 <p>
-  <mixer person="Ulf Juergens" data="name"/> and
-  <mixer person="Gerhard Erich Roehrle" data="name"/>.
+  Ulf Juergens and
+  {% include namelink.html manual="Gerhard Erich Roehrle" text="Gerhard Erich Roehrle" %}.
```
Comment (wucas) [inline Gap3/Packages3/mop.html]: Why is this with a link?

File: Gap3/Packages3/pcqa.html [hunk 2]
```diff
@@ -52,7 +52,6 @@ <h3>
   Contact address
 </h3>
 <p>
-  <mixer person="Eddie Lo" data="contact"/>
+  Eddie Lo<br />email: <a href="mailto:elo@campus.hpu.edu">elo@campus.hpu.edu</a><br />
```
Comment (wucas) [inline Gap3/Packages3/pcqa.html]: Can you check if that looks right?

File: Gap3/Packages3/specht.html [hunk 3]
```diff
@@ -18,7 +18,7 @@ <h3>
   Author
 </h3>
 <p>
-  <mixer person="Andrew Mathas" data="name"/>.
+  {% include namelink.html manual="Andrew Mathas" text="Andrew Mathas" %}.
```
Comment (wucas) [inline Gap3/Packages3/specht.html]: Again normally Authors were without link

File: Gap3/Packages3/weyl.html [hunk 4]
```diff
@@ -17,7 +17,7 @@ <h3>
   Author
 </h3>
 <p>
-  <mixer person="Meinolf Geck" data="name"/>.
+  {% include namelink.html manual="Meinolf Geck" text="Meinolf Geck" %}.
```
Comment (wucas) [inline Gap3/Packages3/weyl.html]: Same here

File: Gap3/Packages3/xmod.html [hunk 5]
```diff
@@ -1,14 +1,14 @@
...
+layout: gap3
+---
 
 <h3>
   The XMod package
 </h3>
 <p>
   Share package since release 3.4, about December 1996,
-  communicated by <mixer person="Derek Holt" data="name_city"/>.<br />
+  communicated by {% include namelink.html manual="Derek Holt" text="Derek Holt" %}.<br />
```
Comment (wucas) [inline Gap3/Packages3/xmod.html]: Here too

File: Gap3/Packages3/mop.html (hunk 1 above)
Comment (AnnaKDS) [reply Gap3/Packages3/mop.html]: Every time people appeared in the addresses.yml file I used their contact information as a link.

File: Gap3/Packages3/pcqa.html (hunk 2 above)
Comment (AnnaKDS) [reply Gap3/Packages3/pcqa.html]: That is just copied from the original webpage.

File: _layouts/gap3.html [hunk 6]
```diff
@@ -0,0 +1,10 @@
+---
+layout: start
+---
+<table><tr>
+<td class="gap3bar">This is a page on GAP&nbsp;3, which is
+still available, but no longer supported. The present version is
+GAP&nbsp;4&nbsp;&nbsp;(See&nbsp; <a href="/Gap3/status3.html">
+Status&nbsp;of&nbsp;GAP&nbsp;3)</a>.</td></tr></table>
```
Comment (fingolfin) [inline _layouts/gap3.html]: I took the liberty of adding this to the layout, and also removed a cache file that was committed by accident

File: Gap3/Packages3/mop.html (hunk 1 above)
Comment (fingolfin) [reply Gap3/Packages3/mop.html]: but Ulf Juergens isn't a link (and it wasn't before, as `data="name"` was used). yet `Gerhard Erich Roehrle` is a link now, even though it also used `data="name"` before. Why this change? I think it should stay as it was, i.e. `Gerhard Erich Roehrle` just a name

Comment (AnnaKDS) [reply Gap3/Packages3/mop.html]: Alright, I guess that it was just me who did it this way. I'll change it back.

File: Gap3/Packages3/pcqa.html (hunk 2 above)
Comment (wucas) [reply Gap3/Packages3/pcqa.html]: Okay still lets check that the result is the same

File: Gap3/Packages3/mop.html (hunk 1 above)
Comment (User141) [reply Gap3/Packages3/mop.html]: I already changed it.

File: Gap3/Packages3/pcqa.html (hunk 2 above)
Comment (User141) [reply Gap3/Packages3/pcqa.html]: It does look the same.

File: Gap3/Packages3/specht.html (hunk 3 above)
Comment (User141) [reply Gap3/Packages3/specht.html]: Done.

File: Gap3/Packages3/weyl.html (hunk 4 above)
Comment (User141) [reply Gap3/Packages3/weyl.html]: Done.

File: Gap3/Packages3/mop.html (hunk 1 above)
Comment (AnnaKDS) [reply Gap3/Packages3/mop.html]: Thanks!

File: Gap3/Packages3/xmod.html (hunk 5 above)
Comment (User141) [reply Gap3/Packages3/xmod.html]: Done.

**Comment by User141** [comment]
> @User141 you just removed all my fixes with your force push. OK, we'll try it differently, I'll submit my changes to your repo with a PR of my own ;-)

I'm sorry. 🙈

**Comment by fingolfin** [comment]
@User141 no worries. please see https://github.com/User141/GapWWW/pull/1

**Pull request merged**